test/core/Graphics.js

```diff
@@ -116,6 +116,845 @@ describe('PIXI.Graphics', function ()
 
             expect(graphics.currentPath.shape.points).to.deep.equal([0, 0, 10, 0]);
         });
+
+        describe('lineJoin', function ()
```

Comment:
This function needs to be wrapped in withGL( like the other functions that use webgl, so that it is ignored by platforms that don't support it.

Reply:
oh snap... i totally overlooked that... will fix asap... TY